README.md

```diff
@@ -89,6 +89,7 @@ This config will be interpreted in the following way:
 - [No Useless Passive](./docs/rules/no-useless-passive.md)
 - [Prefer Observers](./docs/rules/prefer-observers.md)
 - [Require Passive Events](./docs/rules/require-passive-events.md)
+- [Role Supports ARIA Props](./docs/rules/role-supports-aria-props.md)
```

Comment:
Can we move this under `Accessibility-focused rules`?

Reply:
Sure; done in https://github.com/github/eslint-plugin-github/pull/362/commits/ca32268e293048fcc89b15cdc054f2693a02c4ed

Sidebar: Contra our convention, I didn’t prefix this rule with `a11y-`. I wanted to emphasize its connection with the upstream `role-supports-aria-props` rule. It‘s a drop-in replacement.